_typos.toml

```diff
@@ -1,14 +1,19 @@
 # https://github.com/crate-ci/typos#false-positives
 [default]
-extend-ignore-identifiers-re = [
-  "succesfully_destroyed"
-]
 
 [default.extend-identifiers]
 
 [default.extend-words]
 rememberable = "rememberable"
 
+[type.md]
+extend-exclude = [
+  "!UPGRADING.md"
+]
```

Comment:
is this really needed?

Reply:
Hmm I don't recall now if I tested it after adding or not. My understanding of the config was I can add a specific (Markdown) type check with its own settings but we only want to ignore the typo in this single file since it's intentional. I don't believe the config has a way to support that so I guess my thought process was **to exclude anything that isn't the UPGRADING.md file**. Feel free to update and test it locally if you want. I do know the other configurations will still report typos in any of EN based configs (config/locales/en*.yml) so we should have decent coverage there.

Reply:
Hmm I didn't want to ignore the UPGRADING.md file but that may be easiest for now. We can remove the `[type.md]` block then and just update `[files]` list to include the upgrading guide.